doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -283,7 +281,7 @@ on the shape. In particular, the following geometries can be defined so far:</p>
 
 </td></tr>
 <tr><td style="padding-right: 40px; width: 140px;">attrs</td>
-    <td>List of comma separated attributes to include in the entities of the response. If this parameter is not
+    <td>Comma-separated subset of the attributes to be included in the response. If this parameter is not
```

Comment:
Comma-separated list of attribute names which values will be included in the response.